patcher/patcher.go

```diff
@@ -116,6 +116,51 @@ func PatchProc(pid uint32, moduleName string, offsetAddr uintptr, old []byte, ne
 
 	var offset = moduleInfo.LPBaseOfDll + offsetAddr
 
+	// Information about the contents of the memory to read
+	var memoryBasicInfo struct {
+		BaseAddress       uintptr
+		AllocationBase    uintptr
+		AllocationProtect uint32
+		PartitionId       uint32
+		RegionSize        uint64
```

Comment:
`RegionSize` should be a `uintptr` (https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/1dc2ff19-6fef-4c5f-b4fd-afbc2557fd81)